lib/puppet/provider/augeas/augeas.rb

```diff
@@ -103,7 +103,7 @@ def parse_commands(data)
             inSingleTick = !inSingleTick if ch == "'"
             inDoubleTick = !inDoubleTick if ch == "\""
             fail(_("unmatched [")) if nbracket < 0
-          end until ((nbracket == 0 && !inSingleTick && !inDoubleTick && (ch =~ /\s/)) || sc.eos?)
+          end until ((nbracket == 0 && !inSingleTick && !inDoubleTick && (ch =~ /\s/)) || sc.eos?) # rubocop:disable Lint/Loop
```

Comment:
Should we change this to` loop do end` since is similar to `begin end until`?

Reply:
I didn't feel like changing it because the provider is removed from puppet in 6.0 and I didn't want to break anything.